pages/categories/downstream.md

```diff
@@ -12,7 +12,9 @@ Downstream emissions are related to GHG Protocol Scope 3.
 
 {% include linkedHeading.html heading="End-User Devices" level=2 %}
 
-This considers the emissions generated from the electricity consumption of devices such as desktops, laptops, tablets, and mobile phones that utilise the products or services provided by an organisation. It is important to account for the differences in energy efficiency across various devices accessing these products and services. Typically, a smartphone uses less energy than a laptop, and a laptop uses less energy than a desktop. Understanding the [energy use of devices](/information/lifecycle/usage) can then be used to estimate carbon emissions by considering the source of the energy ([Carbon Intensity](/glossary#carbon-intensity)) used to power or charge that device. Finally, end-user devices should have their embodied carbon estimated. This should take into account the product’s life span to consider the entire [Product Life Cycle](/glossary#product-life-cycle). [Read more about hardware lifecycle emissions](/information/lifecycle).
+This considers the emissions generated from the electricity consumption of devices such as desktops, laptops, tablets, and mobile phones that utilise the products or services provided by an organisation. It is important to account for the differences in energy efficiency across various devices accessing these products and services. Typically, a smartphone uses less energy than a laptop, and a laptop uses less energy than a desktop. Understanding the [energy use of devices](/information/lifecycle/usage) can then be used to estimate carbon emissions by considering the source of the energy ([Carbon Intensity](/glossary#carbon-intensity)) used to power or charge that device. Product metrics and web analytics provide valuable information for understanding the number of users, the types of devices the end-users use, where they are located, and the time spent using a product or service. This information is crucial for estimating downstream end-user emissions, and identifying the possible improvements that could be made.
+
+The [embodied carbon of end-user devices](/glossary#product-life-cycle) is not required for attributional accounting of indirect scope 3 emissions. However, there is a consequential effect that organisations should consider when releasing digital products and services. If the software requires significantly powerful hardware or requires the latest operating systems, for example, this could force end-users to replace their hardware and devices to be able to use the product. This carries a downstream consequence of increasing the embodied emissions of end-user devices, as well as increasing waste. [Read more about hardware lifecycle emissions](/information/lifecycle).
```

Comment:
Should this be linking to the embodied carbon glossary definition?

Reply:
yup... my bad

Reply:
I meant to link it to one of your pages.

Reply:
fixed below.

Reply:
looks good to me!